usaspending_api/references/serializers.py

```diff
@@ -75,7 +75,6 @@ class Meta:
 
 class RefProgramActivityBriefSerializer(serializers.ModelSerializer):
     prefetchable = False
-    id = serializers.IntegerField(source='ref_program_activity_id')
```

Comment:
Do we want to serialize the new `id` field to replace the serialization of `ref_program_activity_id`, or do we just not need it?